<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceEditor</class>
 <widget class="QWidget" name="SourceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="toolTip">
          <string>Open another source code file</string>
         </property>
         <property name="text">
          <string>&amp;Open</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+O</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="toolTip">
          <string>Save the current source file</string>
         </property>
         <property name="text">
          <string>&amp;Save</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsButton">
         <property name="toolTip">
          <string>Save As the current source file</string>
         </property>
         <property name="text">
          <string>Save As</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+S</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="toolTip">
          <string>Close editor</string>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+W</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
